File: app/src/main/java/com/neome/java/api/core/deeplink/msg/MsgDeeplinkHtml.java
// neome.ai API - do not change
//

package com.neome.java.api.core.deeplink.msg;

import com.neome.java.api.meta.base.Types.EnumDeviceType;
import com.neome.java.api.meta.base.Types.EnumFormExportType;
import com.neome.java.api.nucleus.base.msg.Msg;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class MsgDeeplinkHtml extends Msg
{
  public String deeplinkCode;

  @Nullable
  public String deviceName;

  @Nullable
  public EnumDeviceType deviceType;

  @Nullable
  public EnumFormExportType exportType;

  @Nullable
  public Long height;

  @Nullable
  public Long width;
}

File: app/src/main/java/com/neome/api/core/deeplink/msg/MsgDeeplinkHtml.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.core.deeplink.msg

import com.neome.api.meta.base.Types.EnumDeviceType
import com.neome.api.meta.base.Types.EnumFormExportType
import com.neome.api.nucleus.base.msg.Msg

interface MsgDeeplinkHtml : Msg
{
  val deeplinkCode: String
  val deviceName: String?
  val deviceType: EnumDeviceType?
  val exportType: EnumFormExportType?
  val height: Long?
  val width: Long?
}